src/WORKSPACE

```diff
@@ -316,3 +322,30 @@ bind(
     name = "six",
     actual = "@six_archive//:six",
 )
+
+# pybind11 for Python bindings
+# quval's repo for pybind11 bazel rules has a fix for hermetic python interpreter
```

Comment:
some more info (ex. link to an issue explaining the problem) would be helpful for folks in the future